package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -57,16 +88,20 @@ class LinearDensity(AnalysisBase):
     ----------
     results.x.dim : int
            index of the [xyz] axes
-    results.x.pos : numpy.ndarray
-           mass density in [xyz] direction
-    results.x.pos_std : numpy.ndarray
+    results.x.mass_density : numpy.ndarray
+           mass density in g/cm^3 in [xyz] direction
+    results.x.mass_density_stddev : numpy.ndarray
            standard deviation of the mass density in [xyz] direction
-    results.x.char : numpy.ndarray
-           charge density in [xyz] direction
-    results.x.char_std : numpy.ndarray
+    results.x.charge_density : numpy.ndarray
+           charge density in (e * mol)/cm^3 in [xyz] direction
```

Comment:
You could use math syntax here

```suggestion
           charge density in :math:`\mathrm{e \cdot mol \cdot cm^{-3}` in [xyz] direction
```